docs/plugins/discogs.rst

```diff
@@ -99,7 +99,7 @@ Here are two things you can try:
 * Make sure that your system clock is accurate. The Discogs servers can reject
   your request if your clock is too out of sync.
 
-Support for matching singleton tracks using ``-s`` is in progress.
-If this is not working well, try the ``--group-albums`` option in album import mode.
+Matching tracks by Discogs ID is not yet supported. The ``--group-albums``
+option in album import mode provides an alternative to singleton mode for autotagging tracks that are not in album-related folders.
```

Comment:
Oh! I completely forgot the fact that `-s` would not work if one would try to supply a Discogs ID with `-S`. I didn't test it yet but what will happen? Ugly error or nothing?

Thanks for mentioning that it doesn't work yet. Very good!

Reply:
I get something like this if I try to use IDs

```
Enter recording ID: r3508566  # the ID for album containing song
No matching records found
[S]kip, Use as-is, ...
```

which is not insightful, but also perfectly reasonable. I think documentation that this feature isn't implemented is fine for now, I don't think changing the interface to display this fact would be worthwhile.